src/assets/translations/en/main.json

```diff
@@ -240,6 +240,18 @@
           "learnMore": "Learn More"
         }
       },
+      "osmosis": {
+        "staking": {
+          "overview": {
+            "rewards": "The rewards you earn will accrue automatically.",
+            "cta": "Start Staking",
+            "learnMore": "Learn More",
+            "withdrawals": "Unstaking",
+            "emptyWithdraws": "Pending unstaking requests will show here.",
+            "availableDate": "available %{date}"
+          }
+        }
+      },
```

Comment:
Unused and Cosmos is fine since we should've never duplicated the two in the first place

```suggestion
```